package.json

```diff
@@ -23,6 +23,9 @@
     },
     "private": true,
     "dependencies": {
+        "@dnd-kit/utilities": "^3.2.1",
+        "@headlessui/react": "^1.7.16",
+        "@heroicons/react": "^2.0.18",
         "react-hook-form": "^7.45.1",
         "react-icons": "^4.10.1"
```

Comment:
Can we get dependencies installed for the client, in the package.json within the client not on the root package.json.
And perhaps with only need one icon library?

Reply:
Removed from root package.json:  "@dnd-kit/utilities": "^3.2.1",
        "@headlessui/react": "^1.7.16",
        "@heroicons/react": "^2.0.18",
   Dnd-kit and headless moved to client package.json.